coding/sha1.cpp

```diff
@@ -1,14 +1,13 @@
 #include "coding/sha1.hpp"
+#include "coding/base64.hpp"
 
 #include "coding/internal/file_data.hpp"
 #include "coding/reader.hpp"
 
 #include "base/assert.hpp"
 #include "base/logging.hpp"
-#include "base/macros.hpp"
 
-#include "3party/liboauthcpp/src/SHA1.h"
-#include "3party/liboauthcpp/src/base64.h"
+#include <boost/uuid/detail/sha1.hpp>
```

Comment:
Is it header-only? Does its output exactly match the output of the previous implementation?

Reply:
Had to do some byte-swapping, but now the output is the same.
Yes, it's header-only.